restcomm/restcomm.sms/src/main/java/org/restcomm/connect/sms/smpp/SmppMessageHandler.java

```diff
@@ -150,6 +150,7 @@ public void onReceive(Object message) throws Exception {
             if (smsMessage == null) {
                 logger.warning("responseMessageId=" + dLRPayload.getId() + " was never received! ");
             } else {
+                smsMessage.setSmppMessageId(null);
```

Comment:
maybe add a meaningful commetn here, setting a filed with null its not very usual, lets make it clear, this is really required for smpp message id reusal

Reply:
agree